devtools/server/actors/replay/react-devtools/hook.js

```diff
@@ -396,6 +398,8 @@ function installHook(target) {
   }
 
   function onCommitFiberUnmount(rendererID, fiber) {
+    window.__RECORD_REPLAY_REACT_DEVTOOLS_HOOK__("commit-fiber-unmount");
```

Comment:
Just so we're clear, is this a random thing unrelated to the object ID stuff?

Reply:
Yeah, as stated above this PR folds in some changes for client side react devtools, which will be the only user of the persistent IDs in the near future.